app/components/SyncError.jsx

```diff
@@ -48,55 +52,20 @@ class SyncError extends React.Component {
         return (
 
             <div className="grid-frame vertical">
-
-                <div className="grid-container text-center" style={{paddingTop: "5rem", maxWidth: "40rem"}}>
+                <div className="grid-container text-center" style={{paddingTop: "5rem", maxWidth: "90%", overflowY: "auto"}}>
```

Comment:
It looks weird on a wide screen, with the scrollbar indented from the right: 

![image](https://user-images.githubusercontent.com/6890015/30314097-eeef31de-979f-11e7-9f1d-2b6dc9cc7cec.png)